azure-pipelines.yml

```diff
@@ -154,10 +169,12 @@ jobs:
           node -v
           npm -v
         displayName: 'Print versions'
-      - task: Npm@1
+      - task: CacheBeta@0
         inputs:
-          command: custom
-          customCommand: ci
+          key: npm | $(Agent.OS) | package-lock.json
+          path: $(npm_config_cache)
+        displayName: 'Cache npm'
+      - script: npm ci
         displayName: 'Install dependencies'
       - script: npm run test:coverage -- --ci --reporters="default" --reporters="jest-junit"
```

Comment:
just for information - for jest in azure pipelines ci flag is unnecessary

Reply:
sometimes `is-ci` buggy for some platforms and CIs (very very very rare), so I always preferred to explicitly use flag to avoid problems

Reply:
But for our case you are right